Example/ios/Podfile

```diff
@@ -7,6 +7,7 @@ require Pod::Executable.execute_command('node', ['-p',
 
 platform :ios, min_ios_version_supported
 prepare_react_native_project!
+ENV['RCT_NEW_ARCH_ENABLED'] = '0'
```

Comment:
Add comment that since 0.76.0 new arch is disabled by default and we need to explicitly turn it off here.

Reply:
Changed in: https://github.com/software-mansion/react-native-screens/pull/2407/commits/5d6718bcd725abf90c185d5a83f114fc6db98725